Dockerfile.apk

```diff
@@ -19,28 +18,25 @@ COPY kong.apk.tar.gz /tmp/kong.apk.tar.gz
 
 RUN set -ex; \
     apk add curl tar gzip ca-certificates; \
-    arch="$(apk --print-arch)"; \
-    case "${arch}" in \
-      x86_64) export ARCH='amd64'; KONG_SHA256=$KONG_AMD64_SHA ;; \
-      aarch64) export ARCH='arm64'; KONG_SHA256=$KONG_ARM64_SHA ;; \
-    esac; \
+    export ARCH='amd64'; \
     if [ "$ASSET" = "remote" ] ; then \
       curl -fL "https://download.konghq.com/gateway-${KONG_VERSION%%.*}.x-alpine/kong-${KONG_VERSION}.${ARCH}.apk.tar.gz" -o /tmp/kong.apk.tar.gz \
       && echo "$KONG_SHA256  /tmp/kong.apk.tar.gz" | sha256sum -c -; \
     fi \
     && tar -C / -xzf /tmp/kong.apk.tar.gz \
-    && apk add --no-cache libstdc++ libgcc perl tzdata libcap zlib zlib-dev bash \
+    && apk add --no-cache libstdc++ libgcc perl tzdata libcap zlib zlib-dev bash yaml \
```

Comment:
What is this one needed for? Is it a tests-only dep?

Reply:
I noticed this as well, if you don't add yaml package, you will got this error
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
INFO: DIR does not exist: /vault/secrets
2023/03/08 19:32:48 [warn] 1#0: the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /usr/local/kong/nginx.conf:6
nginx: [warn] the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /usr/local/kong/nginx.conf:6
2023/03/08 19:32:48 [error] 1#0: init_by_lua error: /usr/local/share/lua/5.1/luarocks/loader.lua:104: error loading module 'yaml' from file '/usr/local/lib/lua/5.1/yaml.so':
	Error loading shared library libyaml-0.so.2: No such file or directory (needed by /usr/local/lib/lua/5.1/yaml.so)
stack traceback:
	[C]: in function 'a_loader'
	/usr/local/share/lua/5.1/luarocks/loader.lua:104: in function </usr/local/share/lua/5.1/luarocks/loader.lua:101>
	[C]: in function 'require'
	/usr/local/share/lua/5.1/lyaml/init.lua:34: in main chunk
	[C]: in function 'require'
	/usr/local/share/lua/5.1/kong/db/declarative/init.lua:2: in main chunk
	[C]: in function 'require'
	/usr/local/share/lua/5.1/kong/init.lua:77: in main chunk
	[C]: in function 'require'
	init_by_lua:2: in main chunk
```